dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -131,8 +133,11 @@
     private long id;
 
     @Column
+    @Deprecated(since = "1.14.0")
     @ApiModelProperty(value = "This is the name of the author stated in the Dockstore.cwl", position = 1)
+    @Schema(description = "This is the name of the author stated in the Dockstore.cwl", deprecated = true)
```

Comment:
I know you copied the existing message to improve the OpenApi doc, but `...stated in the Dockstore.cwl` is not correct (not just CWL, not necessarily named Dockstore.cwl/wdl, may come from .dockstore.yml,...). No good deed goes unpunished. :)